documentation/service-rules/ecf1/payment-engine.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Lead provider API
+---
+
+Learn about how the payment engine works once it receives declarations from lead providers using the Early Career Framework API.
+
+These are the steps
+
+1. Provider submits a declaration.
+2. The system validates data.
+3. If invalid, errors are flagged up with corrective action suggestions.
+4. If valid, the declaration is created and linked to an output fee calculation.
+5. Service fees and banding are determined and applied.
+6. Final payments are calculated and displayed for contract managers.
+
+## Contents
+
+[Lead providers submitting a declaration](#lead-providers-submitting-a-declaration)
+
+[Lead providers reviewing submitted declarations](#lead-providers-reviewing-submitted-declarations)
+
+[Lead providers checking which statement a declaration has been linked to](#lead-providers-checking-which-statement-a-declaration-has-been-linked-to)
+
+[Lead providers checking whether a declaration has been successfully submitted and processed](#lead-providers-checking-whether-a-declaration-has-been-successfully-submitted-and-processed)
+
+[Working with participant bands](#working-with-participant-bands)
+
+[Output Fee vs. Service Fee](#output-fee-vs-service-fee)
+
+[Contract manager logging in and viewing the bandings](#contract-manager-logging-in-and-viewing-the-bandings)
+
+## Lead providers submitting a declaration
```

Comment:
Having another look, we still have a lot of duplication regarding declarations, could we please have this in one place, as in one place for all declaration logic, and one for the payment engine, happy to chat. Let me know your thoughts :)